.gitignore

```diff
@@ -4,6 +4,7 @@
 .idea
 .rbenv-gemsets
 .rbenv-version
+.rspec_results
```

Comment:
I added this so I could use `--next-failure` and `--only-failures` when running `rspec`, which I use a lot.

Reply:
Nice. I use `--only-failures` on other projects, but I didn't know about `--next-failure` :+1: Thanks